README.md

```diff
@@ -67,6 +68,33 @@
 - [NEPs](https://github.com/neo-project/proposals) - NEPs describe standards for the NEO platform.
 - [Network Protocol](http://docs.neo.org/en-us/node/network-protocol.html) - Explanation of how NEO nodes communicate.
 
+## Exchanges
+
+- [Anycoindirect](https://anycoindirect.eu/en/buy/neo) - A Dutch exchange that accepts iDEAL payments.  
+*English, Dutch, German, French, Spanish and Italian interfaces*
+- [Binance](https://www.binance.com/tradeDetail.html?symbol=NEO_BTC) - A Hong Kong-based exchange that allows trading with BTC. Returns GAS to users.  
+*English, Chinese, Korean, Japanese, Russian and Spanish interfaces*
+- [Bitfinex](https://www.bitfinex.com/stats) - A Hong Kong-based exchange where you can trade with BTC.  
+*English, Russian and traditional and simplified Chinese interfaces*
+- [Bittrex](https://bittrex.com/Market/Index?MarketName=BTC-NEO) - A United States-based exchange. Allows trading for BTC, ETH and USDT.  
+*English interface*
+- [Changelly](https://changelly.com/) - A Czech-based exchange that allows trading for various other cryptocurrencies.  
+*English interface*
```

Comment:
If there is only an `English interface`, then I don't think it needs to be stated.